docs/transpile/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Introduction to the transpiler
+description: Introduction to the transpiler
+
+---
+
+
+# Introduction to the transpiler
+
+Transpilation is the process of rewriting a given input circuit to match the topology of a specific quantum device and optimize the circuit instructions for execution on present-day noisy quantum systems.  This documentation covers the tooling and workflows for transpilation available to Qiskit users.  If you're using primitives and are only interested in the default transpilation options, look to the section [Configure runtime compilation for Qiskit Runtime.](../run/configure-runtime-compilation)
+
+A central component of Qiskit, the transpiler is designed for modularity and extensibility. Its central goal is to be able to easily write new circuit transformations (known as transpiler **passes**), and combine them with other existing passes, allowing you to greatly reduce the depth and complexity of your quantum circuits.  Which passes are chained together and in which order has a major effect on the final outcome. This pipeline is determined by `PassManager` and `StagedPassManager` objects.  The `StagedPassManager` will orchestrate the execution one or more `PassMangers` and determine the order in which they are executed, while the `PassManger` object is merely a collection of one or more passes.  Think of the `StagedPassManager` as the conductor in an orchestra, the `PassManagers` as the different instrument sections, and the `Passes` as the individual musicians.  In this way, you have the power to compose hardware efficient quantum circuits and allow you to execute utility scale work while keeping noise manageable.
+
+
+## Transpiler stages
```

Comment:
I think in this section it would be good to link out to your transpiler stages/preset passmanager page and include a CTA for people to go there to understand the stages in more depth